prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -359,7 +359,8 @@ abstract class CoolTest {
             it.organizationName == orgName &&
                 !it.name.contains("FAIL") &&
                 !it.name.contains("BLOBSTORE") &&
-                !it.name.contains("QUALITY")
+                !it.name.contains("QUALITY") &&
+                !it.name.contains("AS2")
         }
```

Comment:
I saw you added AS2 as a fake county in the "State of Ignorance", and that you are using it in AS2TransportTests, but I think you should also add it as a standard part of the end2end and other tests in TestReportStream.

Reply:
I don't have a good AS2 end-point emulator, so I decided against the ene2end test thing. The one I used was pretty flaky. Hard to set up. Couldn't handle more than one request at a time for example.